light/client.go

```diff
@@ -185,7 +185,7 @@ func NewClient(
 	}
 
 	c, err := NewClientFromTrustedStore(chainID, trustOptions.Period, primary, witnesses, trustedStore, options...)
-	if err != nil {
+	if err != nil && !errors.Is(err, ErrEmptyTrustedStore) {
```

Comment:
Why we are lenient with an empty trusted store?

Reply:
because we're bootstrapping the state using hash and height here in case trusted store is empty